diff --git a/node_modules/cypress-svelte-unit-test/dist/utils.js b/node_modules/cypress-svelte-unit-test/dist/utils.js
index f0bab9e..3f19a8e 100644
--- a/node_modules/cypress-svelte-unit-test/dist/utils.js
+++ b/node_modules/cypress-svelte-unit-test/dist/utils.js
@@ -7,9 +7,9 @@ exports.polyfillFetchIfNeeded = exports.injectStylesBeforeElement = exports.clea
 var unfetch_1 = __importDefault(require("unfetch"));
 function checkMountModeEnabled() {
     // @ts-ignore
-    if (Cypress.spec.specType !== 'component') {
-        throw new Error("In order to use mount or unmount functions please place the spec in component folder");
-    }
+    // if (Cypress.spec.specType !== 'component') {
+    //     throw new Error("In order to use mount or unmount functions please place the spec in component folder");
+    // }
 }
 exports.checkMountModeEnabled = checkMountModeEnabled;
 /**
